libbpf-tools/maps.bpf.h

```diff
@@ -4,7 +4,7 @@
 #define __MAPS_BPF_H
 
 #include <bpf/bpf_helpers.h>
-#include <errno.h>
+#include <asm-generic/errno.h>
```

Comment:
right, because it's BPF-only header, it can't include UAPI header, asm-generic/errno.h is the right one, thanks!